package_picks/package-pick-8.19-2024.01+ltac2-debugger.sh

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env bash
+
+###################### COPYRIGHT/COPYLEFT ######################
+
+# (C) 2020..2022 Michael Soegtrop
+
+# Released to the public under the
+# Creative Commons CC0 1.0 Universal License
+# See https://creativecommons.org/publicdomain/zero/1.0/legalcode.txt
+
+###################### CONTROL VARIABLES #####################
+
+# The two lines below are used by the package selection script
+COQ_PLATFORM_VERSION_TITLE="Coq 8.19.2+ltac2-debugger (released Sept 2024)"
+COQ_PLATFORM_VERSION_SORTORDER=98
```

Comment:
98 is the same as 8.19.2.  What should it be here?

Reply:
This is the sort order in the list presented to the user. Official releases start from 1 - additional releases backwards from 100, so I guess 97 might be appropriate. You need to check what is used.

Reply:
FWIW, I can't resolve this conversation.  Presumably security settings.